AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts

```diff
@@ -99,4 +99,11 @@ export class ParticipantsListComponent implements OnInit, OnChanges {
         }
         return interpretee;
     }
+    getCaseName(particpant: ParticipantModel): string {
```

Comment:
I think you should use an array here & includes, like this:
```
['none','observer','panel member'].includes(participant.case_role_name)
```

Reply:
Having said that, I've made some changes in this same area so the point might be moot

Reply:
But, generally, when you're doing multiple string comparisons on the same prop, Array.includes is nice

Reply:
yes, will merge with the refactored version.